### PR TITLE
Add release notes for 10.0.1 and 9.1.1

### PR DESCRIPTION
**Pull request by btashton**
## Summary
Adds release entries for 10.0.1 and 9.1.1 which are security patch only releases.

### CODE REVIEW

File: _releases/10.0.1.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+{% include JB/setup %}
+
+# Apache NuttX-10.0.1 Release Notes
+
+This is a security only patch release.
+
+## Security Issues Fixed In This Release
+Both of the security fixes in this release are part of a larger group of
+vulnerabilities known as AMNESIA:33 that were identified as impacting may IoT
```
Comment (Ouss4) [inline _releases/10.0.1.md:47]:
```suggestion
vulnerabilities known as AMNESIA:33 that were identified as impacting many IoT
```

File: _releases/9.1.1.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+{% include JB/setup %}
+
+# Apache NuttX-9.1.1 Release Notes
+
+This is a security only patch release.
+
+## Security Issues Fixed In This Release
+Both of the security fixes in this release are part of a larger group of
+vulnerabilities known as AMNESIA:33 that were identified as impacting may IoT
```
Comment (Ouss4) [inline _releases/9.1.1.md:47]:
```suggestion
vulnerabilities known as AMNESIA:33 that were identified as impacting many IoT
```

**Comment by Ouss4** [comment]
@btashton Just two small typos.

**Comment by btashton** [comment]
@Ouss4 Thanks for catching that.  Fixed it .

**Review by Ouss4 — APPROVED** [review]

**Pull request merged**